net/freeradius/src/opnsense/mvc/app/models/OPNsense/Freeradius/User.xml

```diff
@@ -63,6 +63,10 @@
                                 <default></default>
                                 <Required>N</Required>
                         </chillispot_bw_max_down>
+                        <sessionlimit_max_session_limit type="IntegerField">
+                                <default></default>
+                                <Required>N</Required>
```

Comment:
Template below wants value as required. So we would need Y here and a sane default? Or a guard in the template...